<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1346</width>
         <height>748</height>
        </rect>
       </property>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1902</width>
          <height>200</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>150</width>
           <height>148</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/pic/logo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="Mun_Name">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>641</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>Municipality name 
slogan</string>
         </property>
        </widget>
        <widget class="QPushButton" name="home_button">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
        <widget class="QPushButton" name="setting_button">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>170</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>1902</width>
          <height>862</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QWidget" name="Setting_page">
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>671</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="update_profile">
             <property name="text">
              <string>Update Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="set_directory">
             <property name="text">
              <string>Set Destination Directory</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="HomePage">
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>540</width>
            <height>80</height>
           </rect>
          </property>
          <property name="text">
           <string>Vital Registration</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>220</y>
            <width>540</width>
            <height>80</height>
           </rect>
          </property>
          <property name="text">
           <string>Citizenship</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>50</y>
            <width>540</width>
            <height>80</height>
           </rect>
          </property>
          <property name="text">
           <string>Sifaris</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>220</y>
            <width>540</width>
            <height>80</height>
           </rect>
          </property>
          <property name="text">
           <string>Relationship Verification</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="sw.qrc"/>
 </resources>
 <connections/>
</ui>
